remove-column/news-column-provider.ts

```diff
@@ -0,0 +1,34 @@
+import { ColumnsProvider, EntityData, ColumnModel, COLUMNS_TOKEN } from "progress-sitefinity-adminapp-sdk/app/api/v1";
+import { Observable, of } from "rxjs";
+import { Injectable } from "@angular/core";
+
+const COLUMN_TO_REMOVE = "LastModified";
+const NEWS_ENTITY_SET_NAME = "newsitems";
+
+/**
+ * A columns provider to provide columns for the list view of the content screen.
+ */
+@Injectable()
+export class NewsColumnProvider implements ColumnsProvider {
```

Comment:
Why do we need a new provider to showcase that we can remove colunms? Lets move the code to the existing DynamicItemIndexColumnsProvider

Reply:
I have added it because I thought we are making a new sample for each new extensibility point.